src/main/java/org/folio/circulation/domain/ServicePointRepository.java

```diff
@@ -31,17 +32,19 @@ public ServicePointRepository(Clients clients) {
   }
 
   public CompletableFuture<Result<ServicePoint>> getServicePointById(UUID id) {
-    log.info("Attempting to fetch service point with id {}", id);
-
     if(id == null) {
-      return completedFuture(succeeded(null));
+      return ofAsync(() -> null);
     }
 
     return getServicePointById(id.toString());
   }
 
   CompletableFuture<Result<ServicePoint>> getServicePointById(String id) {
-    return FetchSingleRecord.<ServicePoint>forRecord("servicepoint")
+    if(id == null) {
```

Comment:
```suggestion
    if (id == null) {
```

I see there is an existing use of `if(` in this file, so maybe we keep it consistent here? I prefer a space after a Java keyword (if not clear from all the suggestions 😄) to easily distinguish it from a method, though, `if` and `while` and `for`, etc., are pretty easily discernible either way for me, maybe not to a Java neophyte or someone who uses google or java style guides.

[Google style](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)
[Java/Sun ancient style](https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html)

Reply:
FOLIO uses the Java/Sun style officially, and that states a preceding space.

I likely need to tweak my IDE to help me remember to do this.